src/Core/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Enable compiler optimization for the `sprintf` function.
 - Avoid calls to spl_object_ methods when computing cache key.
+- Add Sqs AddPermission operation
```

Comment:
You changed the wrong `CHANGELOG` file. This one is fore changes in the `Core` sub package.
You should edit the `./src/Service/Sqs/CHANGELOG`

Reply:
Ah got it.  Right that seems to be passing now apart from the dev check you mentioned.  I've also tested this now on a real account and works as expected.